### PR TITLE
Answer some basic questions on the home page.

### PR DESCRIPTION
**Pull request by bear454**
User feedback from early trials suggested openbare needed some explanation.
This provides some basic context about the app before users log in.

Re: https://github.com/openbare/openbare/issues/11

### CODE REVIEW

File: library/templates/library/home.html [hunk 1]
```diff
@@ -36,6 +36,40 @@
...
+              Any costs associated with the digital assets provided through this
+              <i>openbare</i> instance are handled as follows:
+              <blockquote>{{ host.payment_statement }}</blockquote>
+            {% endif %}
+          </dd>
+          <dt>Is <i>openbare</i> open source?</dt>
+          <dd>
+            Yes! <i>openbare</i> is licensed under the GPLv3 license. You may
+            view with the source, file any bugs or enhancement requests, or
+            help improve <i>openbare</i> through out github project:
```
Comment (rjschwei) [inline library/templates/library/home.html]: No "with"; "our" not "out"

**Comment by rjschwei** [comment]
Only the couple of typos to fix.

File: library/templates/library/home.html (hunk 1 above)
Comment (bear454) [inline library/templates/library/home.html]: Thanks!

**Comment by bear454** [comment]
More commits incoming, though...

**Comment by bear454** [comment]
... okay, I think I'm done now with all the feedback.

**Pull request merged**